src/app.js

```diff
@@ -1 +1,40 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const { terminal } = require('./terminal');
+
+function copyFile(source, destination) {
+  const sourcePath = path.resolve(source);
+  const destPath = path.resolve(destination);
+
+  if (sourcePath === destPath) {
+    console.log('Source and destination are the same. Nothing to do.');
+
+    return;
+  }
+
+  const readStream = fs.createReadStream(sourcePath);
```

Comment:
let's somehow handle this
<img width="640" alt="image" src="https://user-images.githubusercontent.com/37161315/228855999-b2098455-432e-4f95-8ab9-4e54acd9d3fd.png">